www-src/app/replicator/change/change_dispatcher.coffee

```diff
@@ -21,6 +21,7 @@ module.exports = class ChangeDispatcher
     ###
     constructor: (config) ->
         @changeHandlers =
+            "folder": false
```

Comment:
Keeping changesHandlers only for docTypes wich requires changes handlers may be less confusing. (Or at least you should add Notification here ; )